vuework03/src/store/index.js

```diff
@@ -1,25 +1,56 @@
-/** @format */
+import {createStore} from 'vuex';
 
-import { createStore } from 'vuex';
-
-export default createStore({
-	state: {
-		taskTitle: [{ id: 'ID', taskComment: ' コメント ', taskState: ' 状態 ' }],
-		tasks: [],
-		nextTaskId: 0,
-	},
-	mutations: {
-		addTask(state, { comment }) {
-			state.tasks.push({
-				id: state.nextTaskId,
-				comment,
-				status: '作業中',
-				delete: '削除',
-			});
-
-			state.nextTaskId++;
-		},
-	},
-	actions: {},
-	modules: {},
+export default createStore ({
+  state: {
+    tasks: [],
+    sequence: 0,
+  },
+  getters: {
+    tasks: state => {
+      return state.tasks;
+    },
+    sequence: state => {
+      return state.sequence;
+    },
+  },
+  mutations: {
+    createTask (state, {comment}) {
+      const task = {
+        id: state.sequence,
+        comment: comment,
```

Comment:
前回書いていたように、keyと値が一致しているときは
下記のように記述できます。こういった省略記法は積極的に取り入れましょう。
```suggestion
        comment,
```